### ISSUE TITLE
Add wheels for musl linux

### ISSUE THREAD

**Issue by Bruceforce** [sass/libsass-python#391]
Hi, first let me thank you for your awesome work.

Is it possible to add musllinux wheel builds to pypi ([PEP656](https://www.python.org/dev/peps/pep-0656/))?

This would be great for all alpine users since it would simplify and speed up the installation of the libsass package.

**Comment by toabi** [comment]
I guess here's where the wheels are built? https://github.com/sass/libsass-python/blob/main/bin/build-manylinux-wheels

I have no experience with this, but would it be easy to adapt and run a musl container to build musl wheels? There's just no standard OS as for https://github.com/pypa/manylinux as I could see?

**Comment by asottile** [comment]
yeah I haven't seen any standardization on a base image -- we could make our own but ideally someone else would do that work lol

**Comment by toabi** [comment]
Jeah. The cryptography people have their own apparently: https://github.com/pyca/cryptography/blob/main/.github/workflows/wheel-builder.yml#L34

**Comment by asottile** [comment]
yeah they have some special requirements too like rust

**Comment by toabi** [comment]
Just noticed, there are musllinux images there https://quay.io/organization/pypa - here: https://quay.io/repository/pypa/musllinux_1_1_x86_64?tab=tags

**Comment by DeD1rk** [comment]
@asottile I think it's quite sad to see #400 closed. It would be very useful in simplifying and speeding up docker builds. 

So again, would you be open for a PR for building with https://github.com/pypa/cibuildwheel?

**Comment by maybeetree** [comment]
Hello,

I'm seconding this. Musl wheels would be great.

Unfortunately I'm not familiar with either tox or cibuildwheel, so I cannot provide a PR. However, I can confirm that `libsass` compiles without issues on Alpine linux, provided that `gcc`, `g++`, `musl-dev`, and `python3-dev` packages are installed (some of these may be redundant)

Hope this helps.
